.github/workflows/release.yaml

```diff
@@ -0,0 +1,100 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
This means releases are now tagged `vx.y.z` instead of `x.y.z`. I'm OK with that.